virt/storage/virt-storage-config-overview.adoc

```diff
@@ -16,7 +17,7 @@ The following storage configuration tasks are mandatory:
 ifndef::openshift-rosa,openshift-dedicated,openshift-rosa-hcp[]
 xref:../../storage/dynamic-provisioning.adoc#defining-storage-classes_dynamic-provisioning[Configure a default storage class]::
 
-You must configure a default storage class for your cluster. Otherwise, the cluster cannot receive automated boot source updates.
+You must configure a default storage class for the cluster. Otherwise, you cannot import boot source images. `DataVolume` objects (DVs) and `PersistentVolumeClaim` objects (PVCs) that do not explicitly specify a storage class remain in the `Pending` state until you set a default storage class.
```

Comment:
Let's tweak it again a bit :) 
In this context, the boot source images is something that gets imported automatically by OCP Virt: https://docs.redhat.com/en/documentation/openshift_container_platform/4.20/html-single/virtualization/index#virt-creating-vms-from-rh-images-overview

So it's not like the user needs to import them. The user only needs to configure the storage class.